test-framework/common/src/main/java/io/quarkus/test/common/PropertyTestUtil.java

```diff
@@ -22,23 +22,17 @@ public static void setLogFileProperty(String logFileName) {
     }
 
     public static String getLogFileLocation() {
-        return getLogFileLocation(getLogFinalName());
-    }
+        String logFilePath = System.getProperty(LOG_FILE_PATH_PROPERTY);
+
+        if (logFilePath != null) {
+            return logFilePath;
+        }
 
-    private static String getLogFinalName() {
-        return System.getProperty(LOG_FILE_PATH_PROPERTY, FileConfig.DEFAULT_LOG_FILE_NAME);
+        return getLogFileLocation(FileConfig.DEFAULT_LOG_FILE_NAME);
     }
 
     public static Path getLogFilePath() {
-        List<String> logFileLocationParts = getLogFileLocationParts(getLogFinalName());
-        if (logFileLocationParts.isEmpty()) {
-            throw new IllegalStateException("Unable to determine log file path");
-        } else if (logFileLocationParts.size() == 1) {
-            return Paths.get(logFileLocationParts.get(0));
-        } else {
-            return Paths.get(logFileLocationParts.get(0),
-                    logFileLocationParts.subList(1, logFileLocationParts.size()).toArray(new String[0]));
-        }
+        return Paths.get(getLogFileLocation());
```

Comment:
Interesting in your feedback on this particular change as I didn't understand the need for such complexity. But maybe you tried to work around the issue I fixed above.

Reply:
I honestly don't remember, but it certainly could be the case.

If CI is all good, then I have no objections :)

Reply:
That, we will discover soon :).

Reply:
🤞🏼